pkg/config/tlscfg/flags.go

```diff
@@ -70,10 +70,12 @@ func (c ServerFlagsConfig) AddFlags(flags *flag.FlagSet) {
 }
 
 // InitFromViper creates tls.Config populated with values retrieved from Viper.
-func (c ClientFlagsConfig) InitFromViper(v *viper.Viper) Options {
+func (c ClientFlagsConfig) InitFromViper(v *viper.Viper) (Options, error) {
 	var p Options
 	if c.ShowEnabled {
 		p.Enabled = v.GetBool(c.Prefix + tlsEnabled)
+	} else {
+		return p, errors.New("ES_TLS_ENABLED is set to false unable to talk to server(s)")
```

Comment:
I think the point of the issue was to raise an error if enabled=false but **other flags are present**. There's nothing wrong with enabled=false by itself. Also, this is not specific to ES, if you want to refer to the actual property it would be `c.Prefix + tlsEnabled`

Reply:
noted